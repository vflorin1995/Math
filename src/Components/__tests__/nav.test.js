import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import Nav from '../nav';

describe('Jest Snapshot testing suite', () => {
  it('Matches DOM Snapshot', () => {
    const quotes = renderer.create(<Router><Nav /></Router>).toJSON();
    expect(quotes).toMatchSnapshot();
  });
});
