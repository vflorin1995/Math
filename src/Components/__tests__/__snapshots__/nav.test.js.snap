// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite Matches DOM Snapshot 1`] = `
<nav>
  <h1>
    MATH MAGICIANS
  </h1>
  <div>
    <a
      href="#/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="#/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      href="#/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </div>
</nav>
`;
